frontend/src/__mocks__/mockStatus.ts

```diff
@@ -1,11 +1,12 @@
 import { StatusResponse } from '~/redux/types';
+import { createMockToken } from '~/utilities/string';
 
 export const mockStatus = (): StatusResponse => ({
   kube: {
     currentContext: 'opendatahub/admin-user-datahub-redhat-com:6443/admin-user',
     currentUser: {
       name: 'admin-user/api-admin-user-dev-datahub-redhat-com:6443',
-      token: 'sha256~9iRWHnXnoQusPNr11Hda_rqwerfasdfQWERasdfasdf',
+      token: createMockToken(),
```

Comment:
token is not actually used anywhere in the code. So I dont know how much use a createMockToken would be if it always create aa random token. Because then if we did for some reason need to test the token, we wouldn't be able to control its value.

Is there a reason for the mock function here over something like `test-token-value`?